### PR TITLE
Add 1.17.0 support

### PR DESCRIPTION
**Pull request by brendarearden**

### CODE REVIEW

**Comment by deniseschannon** [comment]
https://github.com/rancher/rancher/issues/23088

File: rke/templates/coredns.go [hunk 1]
```diff
@@ -609,3 +609,311 @@ roleRef:
...
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: coredns
+  namespace: kube-system
+data:
+  Corefile: |
+    .:53 {
+        errors
+		health {
```
Comment (superseb) [inline rke/templates/coredns.go]: alignment issue here, should be on the same level as `errors`

File: rke/templates/coredns.go [hunk 2]
```diff
@@ -609,3 +609,311 @@ roleRef:
...
+        {{ $k }}: "{{ $v }}"
+      {{ end }}
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                - key: node-role.kubernetes.io/worker
+                  operator: Exists
+		podAntiAffinity:
```
Comment (superseb) [inline rke/templates/coredns.go]: alignment issue, this should be on same level as `nodeAffinity`

**Comment by dramich** [comment]
LGTM

**Pull request merged**